tools/query_verification/src/test/resources/schema1.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "schema": {
```

Comment:
Do you need this if it only contains 1 property, will this work:

[
  {
    "fields": [],
     "tableReference": {
           ....
      }
  }
]